### PR TITLE
fix(route): 中国工业和信息化部

### LINKED ISSUE #8351: 中国工业和信息化部订阅需要修复
#### 路由地址（不包含参数）

```routes
/gov/miit/zcjd
/gov/miit/wjgs
/gov/miit/zcwj
```

#### 完整路由地址，包含所有必选与可选参数

```fullroutes
https://rsshub.app/gov/miit/zcjd
https://rsshub.app/gov/miit/wjgs
https://rsshub.app/gov/miit/zcwj
```

#### 相关文档地址

- https://docs.rsshub.app/government.html#zhong-guo-gong-ye-he-xin-xi-hua-bu

#### 预期是什么

　　访问工业和信息化部网站，获取相关文章完整内容。

#### 实际发生了什么

　　没有访问正确的页面，没有获取到内容。

### PR DESCRIPTION
**Pull request by chancenju**
## 该 PR 相关 Issue / Involved issue

Close #8351 

## 完整路由地址 / Example for the proposed route(s)

```routes
/gov/miit/wjgs
/gov/miit/zcjd
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [ ] New Route
- [ ] Documentation
  - [ ] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note
已更新

### CODE REVIEW

**Comment by chancenju** [comment]
新手请指教!

**Comment by chancenju** [comment]
救救我把，第一次接触nodejs。16小时过去了，熬不住了。

**Comment by DIYgod** [comment]
How to get the cookie? Is it possible to get it automatically? Or can you add a document for it?

**Review by DIYgod — CHANGES_REQUESTED** [review]

File: lib/config.js [hunk 1]
```diff
@@ -187,6 +187,9 @@ const calculateValue = () => {
         game4399: {
             cookie: envs.GAME_4399,
         },
+        miit: {
+            cookie: envs.miit_cookie,
```
[the remark this answers is not in the document]
Comment (chancenju) [reply lib/config.js:191]: when you first visit the websites. Sorry, I am not an expert on it. I tried and tried, but cannot get the source.

[the remark this answers is not in the document]
Comment (TonyRL) [reply lib/config.js:191]: See #9221

**Comment by TonyRL** [comment]
This route does not require any cookies.

**Pull request merged**